### ISSUE TITLE
IntervalStyle difference on master and segments

### ISSUE THREAD

**Issue by ashwinstar** [greenplum-db/gpdb#1680]
While enabling gpcheckcat at end of ICW, bumped into this issue which gets flagged as inconsistency by gpcheckcat.

Basically, if run the sql "CREAT TABLE inter(a interval DEFAULT '5-7');" with PGOPTIONS="-c intervalstyle=postgres_verbose" it gives

```
SELECT adsrc FROM pg_attrdef WHERE adrelid='inter'::regclass AND adsrc like '%years%';
            adsorb
------------------------------
 '@ 5 years 7 mons'::interval
(1 row)


SELECT adsrc FROM gp_dist_random('pg_attrdef') WHERE adrelid='inter'::regclass and adsrc LIKE '%years%';
           adsrc
----------------------------
 '5 years 7 mons'::interval
 '5 years 7 mons'::interval
(2 rows)
```

So, need to fix this issue of correctly dispatching the config option to segments. As can yield to incorrect results pointed out by Heikki, for example:
```
$ PGOPTIONS="-c intervalstyle=postgres_verbose"  ~/pgsql.master/bin/psql postgres
psql (10devel, server 8.3.23)
Type "help" for help.

postgres=# select i::text from intervaltab ;
       i
----------------
 5 years 7 mons
(1 row)
```
That should return "@ 5 years 7 mons", because intervalstyle=postgres_verbose was used.

**Comment by ivannovick** [comment]
dispatch issue for @gaos1 ?

**Comment by pengzhout** [comment]
@ashwinstar fixed by [1746](https://github.com/greenplum-db/gpdb/pull/1746)

**Comment by ashwinstar** [comment]
Fixed by 8a341fc398f7bfa9ac4be4094dcc4b7f25959a56
